packages/client/src/sync/fetcher/trienodefetcher.ts

```diff
@@ -0,0 +1,578 @@
+import { BranchNode, ExtensionNode, LeafNode, Trie, decodeNode } from '@ethereumjs/trie'
+import {
+  Account,
+  KECCAK256_NULL,
+  KECCAK256_RLP,
+  hexToKeybytes,
+  nibblesToCompactBytes,
+  toBytes,
+} from '@ethereumjs/util'
+import { debug as createDebugLogger } from 'debug'
+import { keccak256 } from 'ethereum-cryptography/keccak'
+import { bytesToHex, equalsBytes, hexToBytes } from 'ethereum-cryptography/utils'
+import { OrderedMap } from 'js-sdsl'
+
+import { Fetcher } from './fetcher'
+
+import type { Peer } from '../../net/peer'
+import type { FetcherOptions } from './fetcher'
+import type { Job } from './types'
+import type { Nibbles } from '@ethereumjs/trie'
+import type { BatchDBOp } from '@ethereumjs/util'
+import type { Debugger } from 'debug'
+
+/**
+ * Packs every two nibbles into a single byte
+ *
+ * @param arr Nibble typed nibble array
+ * @returns Uint8Array typed byte array
+ */
+function nibbleTypeToPackedBytes(arr: Nibbles): Uint8Array {
```

Comment:
we can move these to util/src/encoding.ts

Reply:
Fixed.